### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lykosteng**
add readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,101 @@
...
+- Stackoverflow：[https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
+- Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
+- Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
+- Reddit：[https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
+
+# How Can I Help?
+
+- Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+- To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb)
+- Documentation can always use improvement!
```
Comment (shenli) [inline README.md]: What's mean by this sentence?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,101 @@
...
+- Google Groups：[https://groups.google.com/forum/#!forum/tidb-user](https://groups.google.com/forum/#!forum/tidb-user)
+- Stackoverflow：[https://stackoverflow.com/questions/tagged/tidb](https://stackoverflow.com/questions/tagged/tidb)
+- Twitter: [https://twitter.com/PingCAP](https://twitter.com/PingCAP)
+- Facebook: [https://www.facebook.com/pingcap2015](https://www.facebook.com/pingcap2015)
+- Reddit：[https://www.reddit.com/r/TiDB](https://www.reddit.com/r/TiDB)
+
+# How Can I Help?
+
+- Try [TiDB](https://github.com/pingcap/tidb)! If you like it, star, share, and recommend it. 
+- To dig deeper, read a design doc, e.g. [architecture](https://www.pingcap.com/blog/2017-05-23-perconalive17), [How we build TiDB](https://www.pingcap.com/blog/2016-10-17-how-we-build-tidb)
```
Comment (c4pt0r) [inline README.md]: "read a design doc" seems weird. Or `the` or `our`?

File: README.md (hunk 1 above)
Comment (c4pt0r) [inline README.md]: Documentation is always welcomed!

Comment (Lykosteng) [reply README.md]: Vote for your sentence, I will change it for easy to read.

File: README.md (hunk 2 above)
Comment (Lykosteng) [reply README.md]: Vote for our. Thanks

**Comment by shenli** [comment]
LGTM

**Comment by shenli** [comment]
@c4pt0r PTAL

**Comment by shenli** [comment]
Close the stale  PR.

**Pull request closed without merging**